librz/bin/format/pe/pe.c

```diff
@@ -4048,6 +3876,33 @@ struct rz_bin_pe_lib_t *PE_(rz_bin_pe_get_libs)(struct PE_(rz_bin_pe_obj_t) * bi
 	return NULL;
 }
 
+RzList /* RzBinSymbol */ *PE_(rz_bin_pe_get_methods)(struct PE_(rz_bin_pe_obj_t) * bin) {
+	if (!bin || !bin->clr || !bin->clr->methoddefs) {
+		return NULL;
+	}
+	RzList /* RzBinSymbol */ *methods = rz_list_new();
```

Comment:
should this be 
```suggestion
	RzList /* RzBinSymbol */ *methods = rz_list_newf(rz_bin_symbol_free);
```
?